code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -37,161 +41,370 @@ Doesn't work on other aliens/AI.*/
+	if(!istype(target,/mob))
+		var/list/nearby_mobs = list()
+		for(var/mob/living/M in hearers(1, U))
+			if(isalien(M))
+				continue
+			nearby_mobs += M
+		if(nearby_mobs.len)
+			visible_message_target = pick(nearby_mobs)
+	else
+		visible_message_target = target
+
+	if(visible_message_target)
+		user.visible_message("<span class='alien'>\The [user] spits neurotoxin at [visible_message_target] !</span>", "<span class='alien'>You spit neurotoxin at [visible_message_target] !</span>")
+	else
+		user.visible_message("<span class='alien'>\The [user] spits a salvo of neurotoxin !</span>", "<span class='alien'>You spit out neurotoxin !</span>")
+
+	if(!U || !T)
+		return
+	if(U == T)
+		usr.bullet_act(new /obj/item/projectile/energy/neurotoxin(usr.loc)/*, get_organ_target()*/)
+		return
+
+	var/obj/item/projectile/energy/neurotoxin/A = new /obj/item/projectile/energy/neurotoxin(usr.loc)
+	A.original = target
+	A.target = U
+	A.current = T
+	A.starting = T
+	A.yo = U.y - T.y
+	A.xo = U.x - T.x
+	spawn()
+		A.OnFired()
+		A.process()
+
+/spell/aoe_turf/conjure/choice/alienresin
+	name = "Secrete Resin"
+	desc = "Secrete tough malleable resin."
+	panel = "Alien"
+	hud_state = "alien_resin"
+	override_base = "alien"
+
+	charge_type = Sp_HOLDVAR
+	holder_var_type = "plasma"
+	holder_var_amount = 75
+	insufficient_holder_msg = "<span class='alien'>Not enough plasma stored.</span>"
+
+	spell_flags = IGNORESPACE|IGNOREDENSE|NODUPLICATE
+	full_list = list("Resin Door" = /obj/machinery/door/mineral/resin,"Resin Wall" = /obj/effect/alien/resin/wall,"Resin Membrane" = /obj/effect/alien/resin/membrane,"Resin Nest" = /obj/structure/bed/nest)
+
+/spell/alienacid
+	name = "Corrosive Acid"
+	desc = "Drench an object in acid, destroying it over time."
+	panel = "Alien"
+	hud_state = "alien_acid"
+	override_base = "alien"
+
+	spell_flags = WAIT_FOR_CLICK
+	charge_type = Sp_HOLDVAR|Sp_RECHARGE
+	charge_max = 8 SECONDS
+	holder_var_type = "plasma"
+	holder_var_amount = 200
+	insufficient_holder_msg = "<span class='alien'>Not enough plasma stored.</span>"
 
+	range = 1
 
-/mob/living/carbon/alien/humanoid/proc/corrosive_acid(O as obj|turf in oview(1)) //If they right click to corrode, an error will flash if its an invalid target./N
-	set name = "Corrossive Acid (200)"
+/spell/alienacid/is_valid_target(var/atom/target, mob/user)
+	if(get_dist(user, target) > range) //Shouldn't be necessary but a good check in case of overrides
+		return 0
+	if(!ismob(target) && target.acidable())
+		return 1
+	to_chat(user, "<span class='alien'>You cannot dissolve this object.</span>")
+	return 0
+
+/spell/alienacid/cast(list/targets, mob/user)
+	acidify(targets[1], user)
+
+/mob/living/carbon/alien/humanoid/proc/corrosive_acid(atom/O as obj|turf in oview(1)) //If they right click to corrode, an error will flash if its an invalid target./N
+	set name = "Corrosive Acid (200)"
 	set desc = "Drench an object in acid, destroying it over time."
-	set category = "Alien"
+	set category = null
```

Comment:
Longer but still short answer that I didn't want to type out on my phone: Not technically, but the category setting defaults to "Commands" rather than null